### PR TITLE
User Management backend - Base - Project restructure

### LINKED ISSUE #27: User Management backend - Base
- common models
- DB connection

### PR DESCRIPTION
**Pull request by DoRTaL94**
fixes #27.
- Renamed test folder to tests.
- Renamed flask_init file to app and updated its name inside setup.sh.
- Moved static and templates folders to edison folder.
- Changed network definition to private and removed forwarded ports.
You can read about the reason behind this [here](https://github.com/knowabli/edison/pull/26#discussion_r409288575).

@ahinoamta @RoniRush

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -1,11 +1,8 @@
-FLASK_PORT=5000
-POSTGRESQL_PORT=5432
-
 Vagrant.configure("2") do |config|
+
   config.vm.box = "ubuntu/bionic64"
   config.vm.provision :shell, path: "setup.sh"
-  config.vm.network :forwarded_port, guest: FLASK_PORT, host: FLASK_PORT
- config.vm.network :forwarded_port, guest: POSTGRESQL_PORT, host: POSTGRESQL_PORT
+  config.vm.network "private_network", type: "dhcp"
```
Comment (ifireball) [inline Vagrantfile:5]: you forgot to mention this change in your PR description

File: edison/app.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import edison
+
+from flask import render_template
+
+
+# API description in swagger - https://app.swaggerhub.com/apis/DoRTaL94/UserManagment/1.0.0
```
Comment (ifireball) [inline edison/app.py]: is this comment relevant to this PR?

**Review by ifireball — CHANGES_REQUESTED** [review]

File: edison/app.py (hunk 2 above)
Comment (DoRTaL94) [reply edison/app.py]: No, it would probably be relevent when we will actually add the app routes.
I'll remove it.

File: Vagrantfile (hunk 1 above)
Comment (DoRTaL94) [reply Vagrantfile:5]: Done

Comment (ifireball) [reply Vagrantfile:5]: well, they way you describe it there, tells people **what** you did - which should generally be obvious from the code, but not **why** you did it which is the one thing that code can never tell you on its own...

Comment (DoRTaL94) [reply Vagrantfile:5]: Oh okay I didn't think about it, thank you. I'll rewrite it.

Comment (DoRTaL94) [reply Vagrantfile:5]: @ifireball I referenced your comment in our discussion in the old PR.

Comment (ifireball) [reply Vagrantfile:5]: We'll personally I prefer to write a short summary of what I want people to know, instead of sending then to look for it in a discussion thread, but that is not a reason to block a PR.

**Review by ifireball — APPROVED** [review]
